.github/workflows/copilot-pr-review.yml

```
@@ -0,0 +1,278 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v42
+        with:
+          files: |
+            **/*.yml
+            **/*.yaml
+            **/*.py
+            **/README.md
+            **/_meta/**
+
+      - name: Analyze Changes
+        id: analyze
+        run: |
+          echo "## Changed Intake Formats" > analysis.md
+          echo "" >> analysis.md
+          
+          # Extract vendor/product from changed files
+          vendors=$(echo "${{ steps.changed-files.outputs.all_changed_files }}" | \
+            grep -oP '^[^/]+/[^/]+' | sort -u || echo "")
+          
+          if [ -n "$vendors" ]; then
+            echo "The following intake formats were modified:" >> analysis.md
+            echo "" >> analysis.md
+            echo "$vendors" | while read format; do
+              echo "- \`$format\`" >> analysis.md
+            done
+          else
+            echo "No intake formats were directly modified." >> analysis.md
+          fi
+          
+          echo "" >> analysis.md
+          echo "## Changed File Types" >> analysis.md
+          echo "" >> analysis.md
+          echo "${{ steps.changed-files.outputs.all_changed_files }}" | \
+            grep -oE '\.[^.]+$' | sort | uniq -c | \
+            awk '{print "- " $2 " files: " $1}' >> analysis.md || \
+            echo "- No specific file types detected" >> analysis.md
+          
+          # Save analysis for next step
+          echo "analysis<<EOF" >> $GITHUB_OUTPUT
+          cat analysis.md >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+
+      - name: Check for Required Files
+        id: check-files
+        run: |
+          issues=""
+          warnings=""
+          
+          # Get list of modified vendor/product directories
+          vendors=$(echo "${{ steps.changed-files.outputs.all_changed_files }}" | \
+            grep -oP '^[^/]+/[^/]+' | sort -u || echo "")
+          
+          if [ -n "$vendors" ]; then
+            echo "$vendors" | while read format; do
+              vendor=$(echo "$format" | cut -d'/' -f1)
+              product=$(echo "$format" | cut -d'/' -f2)
+              
+              # Skip if it's a _meta directory
+              if [ "$product" = "_meta" ]; then
+                continue
+              fi
+              
+              # Check for README in vendor directory
+              if [ ! -f "$vendor/README.md" ]; then
+                warnings="$warnings\n- Missing \`$vendor/README.md\`"
+              fi
+              
+              # Check for _meta directory
+              if [ ! -d "$vendor/_meta" ]; then
+                warnings="$warnings\n- Missing \`$vendor/_meta/\` directory"
+              fi
+              
+              # Check for test files
+              if ! find "$format" -name "*test*.py" -o -name "*test*.yml" | grep -q .; then
```

I don't remind we have python tests in format

```suggestion
              if ! find "$format" -name "*test*.yml" | grep -q .; then
```